src/utilities.jl

```diff
@@ -1,7 +1,7 @@
 macro inboundsret(ex)
     return quote
         @inbounds v = $(esc(ex))
-        v
+        return v
```

Comment:
Yes, this was never intended to explicitly return. What I meant was that it "returns" the evaluated expression. I rather have it like it was (but with a rename if you want).